### DISCUSSION TITLE
[Website] Text overflow under Recipes section in introduction page

### DISCUSSION THREAD

**Discussion by Hemanthhari2000** [pmndrs/jotai#2109] [Bug report]
## Summary

Text overflow under Recipes section in introduction page. Please refer the screenshot for desktop and mobile screens.

**Desktop**
![image](https://github.com/pmndrs/jotai/assets/49117799/805f639d-435e-4f57-bbf1-c801e4760940)

**Mobile**
![image](https://github.com/pmndrs/jotai/assets/49117799/b23c9d71-ffb4-495d-8ab5-930cfffea70b)

## Link to reproduction
https://jotai.org/docs/introduction#recipes

## Possible Solution
The above can be solved by adding `break-all` tailwind class to the Link component of TOC in toc.js

**Comment by dai-shi** [comment]
@sandren Can you check it please?
